generic/tests/cfg/boot_vm_in_hugepage.cfg

```diff
@@ -10,8 +10,8 @@
     pre_command_noncritical = yes
     pre_command = "echo 3 > /proc/sys/vm/drop_caches"
     variants:
-        - 2M_hugepage:
-            expected_hugepage_size = 2048
+        - @default:
```

Comment:
Thanks @CongLi . I agree with you that we could set different expected_hugepage_size for different archs.